packages/smooth_app/lib/pages/home_page.dart

```diff
@@ -205,6 +221,7 @@ class _HomePageState extends State<HomePage> {
           ],
         ),
       ),
+
```

Comment:
Please don't add those empty lines: in the dart auto reformatting they will be removed.